code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -199,6 +199,9 @@
 			FindTarget()
 
 /mob/living/simple_animal/hostile/proc/AttackingTarget()
+	if(target == src)
+		to_chat(src,"Why attack yourself?")

```

Comment:
I don't think it's necessary to have this message in here. If it is, it should probably be reformulated so it's not a question.